configs/test/bigquery/datasets.yaml

```diff
@@ -225,3 +225,29 @@ resources:
             type: FLOAT
           - name: engine
             type: STRING
+  - name: cross-pollination-statistics-table
```

Comment:
super nit: I think we use `stats` a lot, so `statistics` can be shortified to it